image/image.go

```diff
@@ -28,3 +37,44 @@ func ValidateDestinationTags(daemon bool, repoNames ...string) error {
 
 	return nil
 }
+
+func VerifyRegistryAccess(regInputs RegistryInputs, keychain authn.Keychain) error {
+	if len(regInputs.ReadableImages()) > 0 {
```

Comment:
I don't think this guard is necessary. If the array is empty, the loop will just be a no-op.

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/5c900400a2e89ae0d9f8a8d0c02243dec574ceba)